packages/fxa-payments-server/src/branding.ftl

```diff
@@ -22,6 +22,7 @@
 # “Accounts” can be localized, “Firefox” must be treated as a brand.
 # 'Firefox Accounts' refers to the service
 project-brand = Firefox Accounts
```

Comment:
This line can be removed as it is no longer being used anywhere.

Reply:
Going to take care of this with removal of old strings task.